### PR TITLE
added user IP in TCPIP menu

### PR DESCRIPTION
**Pull request by gucio321**
Hi ther,
So again, now without fatal code conflicts ;-)
In this PR, I'm adding label with IP address in TCPIP menu. I've also a bit corrected Join Game and Host Game buttons position.

### CODE REVIEW

File: d2game/d2gamescreen/main_menu.go [hunk 1]
```diff
@@ -612,3 +620,25 @@ func (v *MainMenu) onBtnTCPIPCancelClicked() {
 func (v *MainMenu) onBtnTCPIPOkClicked() {
 	v.navigator.ToCharacterSelect(d2clientconnectiontype.LANClient, v.tcpJoinGameEntry.GetText())
 }
+
+// getLocalIP returns local machine IP address
+func (v *MainMenu) getLocalIP() string {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
+
```
Comment (gravestench) [inline d2game/d2gamescreen/main_menu.go:627]: This is assuming that you're on the internet. What happens when you're just on a private LAN...?

**Review by gravestench — APPROVED** [review]

File: d2game/d2gamescreen/main_menu.go (hunk 1 above)
Comment (gravestench) [reply d2game/d2gamescreen/main_menu.go:627]: also, `8.8.8.8:80` is magic; an assumption that the significance of this value is implicit. In 20 years, what will someone looking at this code think? just my 2 cents.

Comment (essial) [reply d2game/d2gamescreen/main_menu.go:627]: What about `LookupHost("localhost")` or something?

Comment (gucio321) [reply d2game/d2gamescreen/main_menu.go:627]: Yah, I also think, that's not a best solution, but I'm not enough in go to do it correct. If you have some time, correct it please or simply I'll remove this function and replace it with TODO

Comment (gravestench) [reply d2game/d2gamescreen/main_menu.go:627]: This should be good for now. I'll make an issue for it.

**Pull request merged**